application/models/CategoryEntity.php

```diff
@@ -11,6 +11,29 @@
  *
  * @author andraavram
  */
-class CategoryEntity {
+  class CategoryEntity extends Entity {
```

Comment:
uncomment this file. it should work. i tested that

Reply:
okay I will uncomment